### PR TITLE
Add curl command to JIRA troubleshooting docs

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Comment by jasonblais** [comment]
Would this command post to a channel?

```
curl -X POST -H 'Content-type: application/json' --data '{"issue":{"fields":{"assignee":{"displayName":"bot@mattermost.com","name":"matterbot"},"description":"asdfasdf","issuetype":{"iconUrl":"https://avatars3.githubusercontent.com/u/13119842","name":"Task"},"priority":{"id":"3","name":"Medium"},"resolution":null,"status":{"id":"10000"},"summary":"Webhook test!"},"key":"TEST-7","self":"http://localhost:8080/rest/api/2/issue/10006"},"user":{"avatarUrls":{"16x16":"https://avatars3.githubusercontent.com/u/13119842=mm&s=16","24x24":"https://avatars3.githubusercontent.com/u/13119842=mm&s=24","32x32":"https://avatars3.githubusercontent.com/u/13119842=mm&s=32","48x48":"https://avatars3.githubusercontent.com/u/13119842=mm&s=48"},"displayName":"bot@mattermost.com","name":"matterbot"},"webhookEvent":"jira:issue_created"}' 'https://<your-mattermost-url>/plugins/jira/webhook?secret=<your-secret>&team=<your-team>&channel=<your-channel>&user_id=admin&user_key=admin'
```

Sorry, I'm just guessing here based on previous scratch notes. Probably better for someone else to attempt. Perhaps @icelander you'd be able to help?

File: source/integrations/jira.rst [hunk 1]
```diff
@@ -90,6 +90,13 @@ Try the following troubleshooting steps:
...
 3. For issue updated events, only status changes when the ticket is reopened, or when resolved/closed are supported. If you'd like to see support for additional events, `let us know <https://mattermost.uservoice.com/forums/306457-general>`_.
 4. If you specified a JQL query in your JIRA webhook page, paste the JQL to JIRA issue search and make sure it returns results. If it doesn't, the query may be incorrect. Refer to the `Atlassian documentation <https://confluence.atlassian.com/jirasoftwarecloud/advanced-searching-764478330.html>`_ for help.
+5. Use a curl command to make a POST request to the webhook URL. If curl command completes with a ``200 OK`` response, the plugin is configured correctly. For instance, you can run the following command
+
+   .. code-block:: text
+
+     curl -v --insecure "https://<your-mattermost-url>/plugins/jira/webhook?secret=<your-secret>&team=<your-team>&channel=<your-channel>&user_id=admin&user_key=admin" --data '{"event":"some_jira_event"}'
+
+   where ``<your-mattermost-url>``, ``<your-secret>``, ``<your-team-url>`` and ``<your-channel-url>`` depend on your setup when configuring the JIRA plugin.
```
[the remark this answers is not in the document]
Comment (jasonblais) [reply source/integrations/jira.rst]: Is my command above correct for posting content to a channel? https://github.com/mattermost/docs/pull/2262#issuecomment-418466514

If we're not sure, we can just add a note it won't result in a post in a channel and figure out the curl command for posting content later.

(I know I should figure out how to do this curl command myself)

**Comment by jasonblais** [comment]
@icelander I saw you shared the following curl command with a customer that posts content to a channel.

Just confirming that's something I can replace the command in my doc with?

```
$ curl -v --insecure "https://<MATTERMOST URL>/plugins/jira/webhook?secret=<SECRET>&team=<TEAM NAME>&channel=<CHANNEL NAME> -d @<FILE NAME>
```

**Comment by jasonblais** [comment]
I don't know if my above curl command posts in a channel, I've just kept the original command and noted no results will be posted in the channel.

File: source/integrations/jira.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwilander) [reply source/integrations/jira.rst]: Looks like it should but don't know for sure unless you try it

**Pull request merged**